fem/pfespace.cpp

```diff
@@ -1348,8 +1333,9 @@ void ParFiniteElementSpace::GetGhostFaceDofs(const MeshId &face_id,
 
    int nv = fec->DofForGeometry(Geometry::POINT);
    int ne = fec->DofForGeometry(Geometry::SEGMENT);
-   int nf = fec->DofForGeometry((nfv == 3) ?
-                                Geometry::TRIANGLE : Geometry::SQUARE);
+   int nf_tri = fec->DofForGeometry(Geometry::TRIANGLE);
+   int nf_quad = fec->DofForGeometry(Geometry::SQUARE);
+   int nf = (nfv == 3) ? nf_tri : nf_quad;
```

Comment:
These are 2 virtual calls instead of 1 before. 😉

Reply:
I need `nf_quad` and `nf_tri` below, this is not just style (but maybe there is a way to do this better, I'll think about it)

Reply:
Oh, OK, I didn't see they are used below.